be/src/storage/page_cache.cpp

```diff
@@ -45,8 +51,31 @@ void StoragePageCache::release_global_cache() {
     }
 }
 
+static void init_metrics() {
+    StarRocksMetrics::instance()->metrics()->register_metric("page_cache_lookup_count", &page_cache_lookup_count);
+    StarRocksMetrics::instance()->metrics()->register_hook("page_cache_lookup_count", []() {
+        page_cache_lookup_count.set_value(StoragePageCache::instance()->get_lookup_count());
+    });
+
+    StarRocksMetrics::instance()->metrics()->register_metric("page_cache_hit_count", &page_cache_hit_count);
+    StarRocksMetrics::instance()->metrics()->register_hook("page_cache_hit_count", []() {
+        page_cache_hit_count.set_value(StoragePageCache::instance()->get_hit_count());
+    });
+
+    StarRocksMetrics::instance()->metrics()->register_metric("page_cache_usage", &page_cache_usage);
+    StarRocksMetrics::instance()->metrics()->register_hook(
+            "page_cache_usage", []() { page_cache_usage.set_value(StoragePageCache::instance()->memory_usage()); });
```

Comment:
Is this computationally expensive? Can I get the value directly from MemTracker?